chart/templates/redis/redis-statefulset.yaml

```diff
@@ -51,10 +51,15 @@ spec:
 {{- with .Values.labels }}
 {{ toYaml . | indent 8 }}
 {{- end }}
-      {{- if .Values.redis.safeToEvict }}
+{{- if or (.Values.redis.safeToEvict) (.Values.redis.podAnnotations) }}
       annotations:
+{{- if .Values.redis.podAnnotations }}
+{{- toYaml .Values.redis.podAnnotations | nindent 8 }}
+{{- end }}
+{{- end }}
```

Comment:
Maybe move this end below the safe to evict block as well? I think it's cleaner wrapping the whole section.